packages/cli/src/commands/up.ts

```diff
@@ -96,16 +102,19 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       return result.urls
     }
 
-    this.log(`Preview environment ${envId} provisioned at ${machine.locationDescription}`)
-
+    this.log(`Preview environment ${envId} provisioned at: ${machine.locationDescription}`)
     ux.table(
       result.urls,
       {
         service: { header: 'Service' },
         port: { header: 'Port' },
         url: { header: 'URL' },
       },
-      this.flags,
+      {
+        ...this.flags,
+        'no-truncate': this.flags['no-truncate'] ?? (!this.flags.output && !this.flags.csv && flags['include-access-credentials']),
+        'no-header': this.flags['no-header'] ?? (!this.flags.output && !this.flags.csv && flags['include-access-credentials']),
```

Comment:
IMO this is redundant, let user decide what they want

Reply:
The problem is that `include-access-credentials` UI is broken, when these flags are on

Reply:
It's also broken when your branch name is too long. Having proper tables according to the tty column width is a feature - we can decide to never implement it but as long as we do, it should work.